assets/blocks/reader-registration/edit.js

```diff
@@ -313,8 +315,8 @@ export default function ReaderRegistrationEdit( {
 										<RichText
 											onChange={ value => setAttributes( { privacyLabel: value } ) }
 											placeholder={ __( 'Terms & Conditions statement…', 'newspack' ) }
-											value={ privacyLabel }
-											tagName="p"
+											value={ privacyLabel || defaultTermsText }
+											tagName={ defaultTermsUrl ? 'a' : 'p' }
```

Comment:
The `tagName` must still be a `p` (with an `<a />` inside) in order for the link to be editable. This strategy also doesn't implement the correct markup:

```html
<p><a href="#">Text</a></p>
```

Reply:
Ah, thanks for highlighting. I wasn't sure what the correct nesting would be for the `RichText` tag.

Reply:
333303b should render the correct markup in the editor.